mod/state-transition/pkg/core/state_processor_genesis_test.go

```diff
@@ -320,9 +261,3 @@ func commonChecksValidators(
 		require.Equal(t, math.Gwei(0), val.EffectiveBalance)
 	}
 }
-
-// in genesis UTs we don't need to verify proposer address
-// (no one proposes genesis), hence the dummy implementation.
-func dummyProposerAddressVerifier(bytes.B48) ([]byte, error) {
-	return nil, nil
-}
```

Comment:
genesis should not really have a proposer but I guess we don't really care here, so I am in favour of the simplification